src/Tribe/Settings_Tab.php

```diff
@@ -87,6 +87,24 @@ class Tribe__Settings_Tab {
 	 */
 	public $priority;
 
+	/**
+	 * Parent tab ID, if any.
+	 *
+	 * @since TBD
+	 *
+	 * @var string $parent
+	 */
+	public $parent = null;
```

Comment:
These are new, so I think it would be better to make the `protected` (or even `private`), and provide `get_*()` methods and `has_*()` methods.